.swiftformat

```diff
@@ -0,0 +1,15 @@
+# format options
+--funcattributes same-line
+--ranges no-space
+
+# rules enabled
+--enable wrapEnumCases
+
+# rules disabled
+--disable andOperator
+--disable redundantPattern
+--disable redundantLet
+--disable wrapMultilineStatementBraces
+
+# min swift format version
+--minversion 0.50.6
```

Comment:
The issue says:

> You should set up linters in Xcode in the build phase:
SwiftLint - version 0.51.0
SwiftFormat - version 0.51.7

Then why minversion here is `0.50.6`?

Reply:
Discuss with the team, everyone has already updated his dev environment, so we agreed to add version  0.51.7 as min version.

Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/24/commits/9749257c6b99d25724eb9730fca446b36ecf3ca3